build.gradle

```diff
@@ -4,12 +4,6 @@ plugins {
   id 'net.researchgate.release' version '2.8.1'
 }
 
-ext {
-  cglibVersion = '3.2.6'
-  configVersion = '1.4.0'
-  junitJupiterVersion = '5.6.0'
-}
-
 group = 'cz.datadriven.utils'
 
 apply plugin: 'idea'
```

Comment:
Can be moved to `plugins` section, maybe?

Reply:
✅